compiler/rustc_passes/src/check_attr.rs

```diff
@@ -2087,7 +2088,32 @@ impl CheckAttrVisitor<'_> {
 
     fn check_macro_export(&self, hir_id: HirId, attr: &Attribute, target: Target) {
         if target != Target::MacroDef {
-            self.tcx.emit_spanned_lint(UNUSED_ATTRIBUTES, hir_id, attr.span, errors::MacroExport);
+            self.tcx.emit_spanned_lint(
+                UNUSED_ATTRIBUTES,
+                hir_id,
+                attr.span,
+                errors::MacroExport::Normal,
+            );
+        } else if let Some(meta_item_list) = attr.meta_item_list() {
```

Comment:
What if it's a `#[macro_export = ".."]`?

Reply:
When testing this:

```
error: malformed `macro_export` attribute input
  --> fake-test-src-base/invalid_macro_export_argument.rs:31:1
   |
LL | #[macro_export = "Not a valid value"] //~ WARN `not_local_inner_macros` isn't a valid `#[macro_export]` argument
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
```

It seems like Rust already gives an error in that case.When testing this:

```
error: malformed `macro_export` attribute input
  --> fake-test-src-base/invalid_macro_export_argument.rs:31:1
   |
LL | #[macro_export = "Not a valid value"] //~ WARN `not_local_inner_macros` isn't a valid `#[macro_export]` argument
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
```

It seems like Rust already gives an error in that case.